### PR TITLE
Add progress bar to Canasta Create

### PR DESCRIPTION
**Pull request by gitmapd**

### CODE REVIEW

**Comment by yaronkoren** [comment]
Note: this is a fix for issue #25.

File: cmd/create/create.go [hunk 1]
```diff
@@ -68,27 +68,36 @@ func NewCmdCreate() *cobra.Command {
 
 // importCanasta accepts all the keyword arguments and create a installation of the latest Canasta.
 func createCanasta(canastaInfo canasta.CanastaVariables, pwd, path, orchestrator string) error {
+	bar := progressbar.Default(100)
```
Comment (bawolff) [inline cmd/create/create.go:71]: This seems to be missing an import

**Review by bawolff — CHANGES_REQUESTED** [review]
When i try to build i get:
```
# github.com/CanastaWiki/Canasta-CLI-Go/cmd/create
cmd/create/create.go:71:9: undefined: progressbar
```

Additionally, I'm not sure this progress bar is meaningfully fine-grained enough. The first time you start Canasta, a lot of the time is going to be spent in the `docker-compose up -d` phase, since all the images have to be downloaded and built, which may be a significant amount of time. I'm not sure that having the progress bar be stuck on 45% is really all that much better than not having a progress bar.

**Comment by yaronkoren** [comment]
A separate approach to this (using spinners instead of a progress bar) was just checked in, in #101.

**Pull request closed without merging**